src/shared/firewall-util-nft.c

```diff
@@ -964,6 +967,46 @@ static int fw_nftables_add_masquerade_internal(
                                               source, source_prefixlen);
 }
 
+int nft_set_element_modify_in_addr_open(
```

Comment:
This is mostly equivalent to `nft_set_element_modify_in_addr()`. I do not think it is necessary to introduce such wrapper.

Reply:
I will consider again in next cycle of review. Feel free to keep it now.

Reply:
Still I do not think such wrapper. At least, log message is duplicated. in this function and in the caller.